front/pages/api/user/index.ts

```diff
@@ -83,6 +84,18 @@ async function handler(
           },
         });
       }
+      if (user.workspaces[0]?.role === "admin") {
+        sendUserOperationMessage({
+          message: `workspace_sid: ${user.workspaces[0]?.sId}; email: [${user.email}]; User Name [${user.firstName} ${user.lastName}].`,
```

Comment:
nit: let's add a verb no ? (like "created")

Reply:
it's not really created, so hard to actually attach a meaningful action to it.
We go through this flow when the user complete their profile after creating a workspace :/